tests/profile/ProfileUiTest.php

```diff
@@ -1,13 +1,12 @@
 <?php
 
-class ProfileUiTest extends PHPUnit_Framework_TestCase
+require_once('AbstractProfileTest.php');
```

Comment:
It's based on convention of directory name = php namespace name.  You also need to define roots so that the autoloader knows where to look, i.e. in the tests directory.  Happy to leave it for now.